### ISSUE TITLE
Missing statistics data from Scratch

### ISSUE THREAD

**Issue by maluta** [LLK/scratch-www#6601]
### Expected Behavior

When accessing https://scratch.mit.edu/statistics/country/ I expect see all stats (projects, users, comments) from my country (in my case Brazil) 

### Actual Behavior

Page loads but some data is missing. 

### Steps to Reproduce

1. Go to https://scratch.mit.edu/statistics/country/

### System Details

**Screenshots**

<img width="1157" alt="Screen Shot 2022-03-04 at 12 53 23" src="https://user-images.githubusercontent.com/23658/156796141-4f7bf5bf-9a30-4e62-a68b-baf5c7a645ab.png">

<img width="323" alt="Screen Shot 2022-03-04 at 12 54 25" src="https://user-images.githubusercontent.com/23658/156796164-bf27bc90-3594-42bb-82dc-1ab364256ae0.png">

**Comment by Chiroyce1** [comment]
> When accessing https://scratch.mit.edu/statistics/country/ I expect see all stats (projects, users, comments) from my country (in my case Brazil)

Actually there seems to be another bug - it shows Brazil for me and a few other Scratchers not from Brazil as well!

**Comment by maluta** [comment]
> Actually there seems to be another bug - it shows Brazil for me and a few other Scratchers not from Brazil as well!

Good point! Since I always accessed this URL from Brazil, I assumed this endpoint (/country) should filter the user's geography.

**Comment by jeffalo** [comment]
Looks like https://scratch.mit.edu/statistics/data/country/monthly/ is returning a 503.

![image](https://user-images.githubusercontent.com/40470736/156830586-00e36be5-aa4f-4353-b9b4-3e543bbafc20.png)

![image](https://user-images.githubusercontent.com/40470736/156830515-ef4502ed-1881-4d84-aa70-48bdf834e448.png)

**Comment by Chiroyce1** [comment]
> Looks like https://scratch.mit.edu/statistics/data/country/monthly/ is returning a 503.

Yep, it seemed to have occurred right after we posted and just before you did.
